src/transaction/log_manager.c

```diff
@@ -3275,16 +3233,6 @@ log_skip_logging_set_lsa (THREAD_ENTRY * thread_p, LOG_DATA_ADDR * addr)
 void
 log_skip_logging (THREAD_ENTRY * thread_p, LOG_DATA_ADDR * addr)
```

Comment:
@wrlawodms There is no difference in the behavior of the log_skip_logging() function before and after the change. It would be nice to complete the fix here for the purpose of this issue. And the removal of the log_skip_logging() completely would be nice to handle other issue because the purpose of this issue is to remove #if 1 or #if 0 macro. Please give me any opinion if you have.

Reply:
@hornetmj The way I see it, removing the `#if 0` part means we are gonna remove `log_skip_logging()`, but I'm not sure about this. In addition, If we remove `log_skip_logging()`, the macro will be removed along.

Reply:
@wrlawodms OK.
@Yechan0815 Please leave a comment that we need to decide whether or not remove log_skip_logging_set_lsa() function at the http://jira.cubrid.org/browse/CBRD-24398.

Reply:
@hornetmj  I added!
Do I need to revert the deletion of macro in function `log_skip_logging ()`?

Reply:
@Yechan0815 Yes, please.